src/js/components/stock-movement-wizard/modals/EditPickModal.jsx

```diff
@@ -26,11 +26,7 @@ import { formatDate } from 'utils/translation-utils';
 const FIELDS = {
   reasonCode: {
     type: SelectField,
-    label: () => (
-      <label htmlFor="reasonCode" className="ml-3 text-center">
-        <Translate id="react.stockMovement.reasonCode.label" defaultMessage="Reason code" />
-      </label>
-    ),
+    label: 'react.stockMovement.reasonCode.label',
```

Comment:
@EWaterman I took a quick look to find a potential reason for that to be broken just now. 
The bug is a bit deeper, hence I am not surprised you had some difficulties in even finding a "make it work" solution.
It got broken by this change: https://github.com/openboxes/openboxes/commit/f55ed0d236a6a3f89a7e6766910b8ac9a48bf5fa#diff-1f44efd686996ca77c130f21050ac254054b889fdd02d38b562605191318babbR49

We pass a function to the `translate`, when it expects us to pass a string. For this particular case, we don't provide the `ariaLabel` prop, so we have a fallback to: `translate(FieldLabel, defaultMessage)`.

I can see that this is the only place where our `label` is a method, so I am fine with this change, but I am not sure if this `<label>` element didn't have some "hidden" behavior for this field, but I assume not (even if it had, we would still wrap this field around `<label>`, as it is wrapped in the `form-utils.jsx` 73th line.